distribution/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -138,3 +138,892 @@ For example, if a Spatial search and Contextual search are the components of a C
+  |                  |        |                       |-------------------------|-------------->|                     |                        |       |        |
+  |                  |        |                       |   ResourceResponse      |               |                     |                        |       |        |
+  |                  |        |                       |<------------------------|---------------|                     |                        |       |        |
+  |Web Service       |        |       ResourceResponse|                         |               |                     |                        |       |        |
+  |     Retrieval Response    |<----------------------|                         :               |                     |                        |       |        |
+  |<-----------------|--------|                       |                         |               |                     |                        |       |        |
+  |                  |        |                       |                         |               |                     |                        |       |        |
+  |                  +---------------------------------------------------------------------------------------------------------------------------------+        |
+  |                                                                                                                                                             |
+....
+
+
+The Catalog Framework optionally provides caching of products, so future requests to retrieve the same product will be serviced much quicker.
+If caching is enabled, each time a retrieve product request is received, the Catalog Framework will look in its cache (default location: `<INSTALL_DIR>/data/productcache`)to see if the product has been cached locally.
+If it has, the product is retrieved from the local site and returned to the client, providing a much quicker turnaround because remote product retrieval and network traffic was avoided.
+If the requested product is not in the cache, the product is retrieved from the Source (local or remote) and cached locally while returning the product to the client.
+The caching to a local file of the product and the streaming of the product to the client are done simultaneously so that the client does not have to wait for the caching to complete before receiving the product.
+If errors are detected during the caching, caching of the product will be abandoned, and the product will be returned to the client.
+
+The Catalog Framework attempts to detect any network problems during the product retrieval, such as long pauses where no bytes are read, implying a network connection was dropped.
+(The amount of time that a "long pause" is defined as is configurable, with the default value being five seconds.)
```

Comment:
When my other change goes through, it can just be linked to the configuration in the managing section.